rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/db/sql/DBSQLFile.kt

```diff
@@ -19,7 +20,7 @@ class DBSQLFile(private val name: String, private val connection: Connection) :
         if (resultSet == null) {
```

Comment:
You could use require here. I personally find it more obvious

Reply:
You're right!